test/execWithOutput.test.js

```diff
@@ -111,3 +112,43 @@ tap.test('passes env vars excluding `INPUT_*` env vars', async t => {
   // Its value will vary by test runner, so just check it is present.
   t.hasProp(envInExec, 'NODE')
 })
+
+tap.test('Invalid arguments inputs should not fail', async t => {
+  const redactedBlankArray = removeConfidentialArguments([]);
+  const redactedUndefinedArray = removeConfidentialArguments(undefined);
+  const redactedNullArray = removeConfidentialArguments(null);
+
+  t.equal(Array.isArray(redactedBlankArray), true)
+  t.equal(Array.isArray(redactedUndefinedArray), true)
+  t.equal(Array.isArray(redactedNullArray), true)
+
+  t.equal(redactedBlankArray?.length, 0)
+  t.equal(redactedUndefinedArray?.length, 0)
+  t.equal(redactedNullArray?.length, 0)
```

Comment:
the conditional operator is unnecessary, you have already tested that it's an array